actors/rozetka-count/src/main.js

```diff
@@ -0,0 +1,115 @@
+import Apify from 'apify';
+
+import { getOrIncStatsValue, migrationFlagGetOrSet } from './tools.js';
+import { handleMainPage, handleCategory, handleProductList,
+    handleRestaurantCategory, handleRestaurantSubcategory,
+    handleClothesCategory } from './routes.js';
+import { LABELS, START_REQUESTS, STATS_KEY } from './consts.js';
+
+const { MAIN_PAGE, CATEGORY, PRODUCT_LIST,
+        RESTAURANT_CATEGORY, RESTAURANT_SUBCATEGORY,
+        CLOTHES_CATEGORY } = LABELS;
+
+const {
+    utils: { log },
+} = Apify;
+
+Apify.main(async () => {
+    const input = await Apify.getInput();
+    const {
+        maxConcurrency = 100,
+        proxyCountryCode,
+        maxRequestsPerCrawl,
+    } = typeof input === 'object' ? input : {
```

Comment:
typeof null === "object". So I would instead do
const input = (await Apify.getInput()) || defaultInput;

It is also more readable

Reply:
Yeah, thanks for noticing! Will change that.